app/services/create_judgement.rb

```diff
@@ -0,0 +1,37 @@
+class CreateJudgement
+
+  def self.call(evaluation, params)
```

Comment:
Service objects are generally non-static classes, you instantiate it with your parameters, and then call `call` without any params. The advantage of that style is that you don't have to pass 'evaluation' and 'params' to every function manually, they can access them as a class variable